CONTRIBUTING.md

```diff
@@ -43,11 +75,45 @@ Liquibase Hub or Liquibase Pro.
 DB migrations should be written to roll back cleanly (exceptions should be discussed with the team).
 
 -------------
-
 # swatch-product-configuration
 
 Directory structure: `$Platform/$Subscription_Name.yaml` (e.g. RHEL/RHEL_for_x86.yaml).
 
 Use camel case for yaml properties that need to be serialized into the `com.redhat.swatch.configuration.registry` object definitions.
 
 The `includedSubscriptions` property only applies to metrics that aren't sourced from prometheus.  It's used to prevent double counting capacity.
+
+-------------
+# Podman installation instructions
+
+Since we're using Testcontainers to set up services during test execution, besides installing podman, you also need to enable podman socket with 
+Docker REST API.
+
+First, you need to install podman. For Linux systems, you need to do:
+
+```shell
+sudo dnf install podman podman-docker
+```
+
+In other systems, follow [the official site guide](https://podman.io/docs/installation) to install podman.
+
+Once installed, you need to enable the podman socket with Docker REST API with systemd (only needs to be done once):
+
+```shell
+systemctl --user enable podman.socket --now
+```
+
+In systems when systemd is not running (WSL etc), instead of the above command, you need to do:
+
+```shell
+podman system service --time=0
+```
+
+Then, you can obtain the path of the socket with the following command:
```

Comment:
We need to explain what to do with the socket (`export DOCKER_HOST=unix:///path/to/podman.sock`).

Reply:
Added